lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -324,6 +324,11 @@ def add_files_to_build_phases(native_target, test_native_targets, app_native_tar
 
               filter_resource_file_references(file_accessor.resources.flatten) do |compile_phase_refs, resource_phase_refs|
                 native_target.add_file_references(compile_phase_refs, nil)
+
+                if target.build_as_static_framework?
+                  resource_phase_refs = resource_phase_refs.select { |ref| Target.resource_extension_compilable?(File.extname(ref.path)) }
```

Comment:
dont we need to apply this to a copy resources script phase input and output paths? Isn't it possible that those two will get out of sync?

Reply:
Please clarify - what could get out of sync with what?

Reply:
Oh this gets added to normal Xcode "Copy Bundle resources" versus the CocoaPods "Copy Resources" phase? If so then yes I dont think the paths will get out of sync

Reply:
@igor-makarov is my question/understanding correct?

Reply:
Yeah, exactly, because a static framework is a framework, it has a location to put them in!

And XIBs and such get compiled by the plain Xcode resource step so later on we just rsync the products.